### PR TITLE
[NPC] Franclorn Forgewright visibility

### LINKED ISSUE #549: [NPC] Franclorn Forgewright is visible to LIVE players.
**Description:** 
The ghost of Franclorn Forgewright is visible to LIVE players.

**Expected behavior:** 
The ghost of Franclorn Forgewright should only be visible to DEAD players.

**Steps to reproduce the problem:**
Visit the central area of Blackrock Mountain alive and go to his tomb.

**Branch:**
Master

**Commit Hash:** 
 f353f9b5bb08cde4d75b13c1c1c33ae35da971b4

### PR DESCRIPTION
**Pull request by Viste**
**Changes proposed:**

-   make Franclorn Forgewright visible only for dead players

**Target branch(es):** 1.x/2.x etc.
master
**Issues addressed:** Closes #549 

**Tests performed:** (Does it build, tested in-game, etc)
test results you can see in issue

thx @Killyana for this fix

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1503198096265931900.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1503198096265931900');
+
+-- make Franclorn Forgewright visible only for dead players
+UPDATE `creature_template` SET `npcflag`=32771, `AIName`='' WHERE `entry`=8888;
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 8888  AND `source_type` = 0;
+UPDATE `creature_template_addon` SET `auras`=10848 WHERE `entry` IN (8888);
```
Comment (ShinDarth) [inline data/sql/updates/pending_db_world/rev_1503198096265931900.sql]: please add a new line at the end of file

Comment (Viste) [inline data/sql/updates/pending_db_world/rev_1503198096265931900.sql]: @ShinDarth done

**Comment by Jagermaister95** [comment]
I tested the fix and works perfectly.
Tested on the azerothcore server demo.

**Comment by BarbzYHOOL** [comment]
so when an issue has "test-needed" it's testable on AC server demo?

**Comment by Yehonal** [comment]
No, PR on staging branch means that is on demo too

Il 21/Ago/2017 00:18, "Barbz" <notifications@github.com> ha scritto:

**Comment by ShinDarth** [comment]
I changed the color to make those 2 label slightly different :dancer:

**Comment by tea91** [comment]
Is this acceptable for Soul of Azuregos?

**Pull request merged**